### PR TITLE
Hide row filters for secrets on service account page

### PR DESCRIPTION
**Pull request by spadgett**
/assign @jhadvig

### CODE REVIEW

File: frontend/public/components/secret.jsx [hunk 1]
```diff
@@ -115,7 +115,11 @@ const SecretsPage = props => {
     createLink: (type) => `/k8s/ns/${props.namespace}/secrets/new/${type !== 'yaml' ? type : ''}`
   };
 
-  return <ListPage ListComponent={SecretsList} canCreate={true} rowFilters={filters} createButtonText="Create" createProps={createProps} {...props} />;
+  // Only show row filters if no props.filters were passed. We want to hide the
+  // filter bar when already filtering secrets for pages like the service
+  // account page.
+  const rf = _.isEmpty(props.filters) ? rowFilters : null;
```
Comment (spadgett) [inline frontend/public/components/secret.jsx]: Debating if this change shouldn't be made in ListPage

/hold

**Comment by jhadvig** [comment]
Don't we want put the Secrets list into a tab ? Similarly as `Role Bindings` are tabbed for `Namespaces` and `Projects`.

File: frontend/public/components/secret.jsx (hunk 1 above)
Comment (jhadvig) [reply frontend/public/components/secret.jsx]: I think moving it down to the ListPage makes sense.

**Comment by spadgett** [comment]
Closing in favor of #223

/close

**Pull request closed without merging**